### DISCUSSION TITLE
Paypal Express Payments Depricated

### DISCUSSION THREAD

**Discussion by alex-pommier** [HotcakesCommerce/hotcakes-commerce-core#444] [Q&A]
I am using Stripe as the gateway for Credit Card processing but would also like to offer paypal payments however reading around I see Paypal Express payments are deprecated and the new options offered from Braintree don't match the required API Username, Password & Signature fields in Paypal Express Payment settings. Is there any work-around?

**Comment by WillStrohl** [comment]
The next release of Hotcakes Commerce (3.7.0) brings with it a completely rewritten replacement for PayPal Pro, PayPal Express, and Stripe.
